returnn/training.py

```diff
@@ -402,6 +403,12 @@ def check_blacklisted_parameters(self, returnn_config):
                 % key
             )
 
+        config_blacklisted_keys = ["cleanup_old_models"]
+        for key in config_blacklisted_keys:
+            assert returnn_config.config.get(key) is None, (
+                "please define %s only as parameter in the post_config" % key
+            )
+
```

Comment:
Would this not be caught by `Config.check_consistency` or, if not, would we rather have this check added to the `check_consistency` method?

Reply:
Yes this would be caught, I just wanted to add it here to remind a user to not put the cleanup parameter in the hashable part.

Reply:
I am all in to have a set of config options defined that should never be hashed and have an explicit check for that.
I just still think as this check is solely about the state of the config, it should live in the config object itself (e.g. the `check_consistency` method, but can be a different place) and can be used by all returnn_config processing jobs.

This is not so important for cleanup, but e.g. `log_verbosity` should not be hashed for training, decoding, prior estimation. So it would be nice to have the check at a central place.